articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,487 @@
+
+1. With Visual Studio, open `1 - Starter Application\src\AzureCafe.sln`.
+
+2. In the Solution Explorer panel, right-click the AzureCafe project file, and select **Manage NuGet Packages**.
+
+    ![The Solution Explorer displays with Manage NuGet Packages selected from the context menu.](./media/asp-net-core/manage_nuget_packages_menu.png "Manage NuGet Packages")
+
+3. Select the **Browse** tab, then search for and select **Microsoft.ApplicationInsights.AspNetCore**. Select **Install**, and accept the license terms. It is recommended to use the latest stable version. Find full release notes for the SDK on the [open-source GitHub repo](https://github.com/Microsoft/ApplicationInsights-dotnet/releases).
+
+    ![The NuGet tab displays with the Browse tab selected and Microsoft.ApplicationInsights.AspNetCore is entered in the search box. The Microsoft.ApplicationInsights.AspNetCore package is selected from a list of results. In the right pane, the latest stable version is selected from a drop down list and the Install button is highlighted.](./media/asp-net-core/asp-net-core-install-nuget-package.png "Install NuGet Package")
+
+4. Keep Visual Studio open for the next section of the article.
+
+## Enable Application Insights server-side telemetry
+
+The Application Insights for ASP.NET Core web applications NuGet package encapsulates features to enable sending server-side telemetry to the Application Insights resource in Azure.
+
+1. From the Visual Studio Solution Explorer, locate and open the **Program.cs** file.
+
+    ![The Visual Studio Solution Explorer displays with the Program.cs highlighted.](./media/asp-net-core/solution_explorer_programcs.png "Program.cs")
+
+2. Insert the following code prior to the `builder.Services.AddControllersWithViews()` statement. This code automatically reads the Application Insights connection string value from configuration. The `AddApplicationInsightsTelemetry` method registers the `ApplicationInsightsLoggerProvider` with the built-in dependency injection container, that will then be used to fulfill [ILogger](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger) and [ILogger\<TCategoryName\>](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.iloggerprovider) implementation requests.
+
+    ```csharp
+    builder.Services.AddApplicationInsightsTelemetry();
+    ```
+
+    ![A code window displays with the preceding code snippet highlighted.](./media/asp-net-core/enable_server_side_telemetry.png "Enable server-side telemetry")
+
+    > [!TIP]
+    > Learn more about [configuration options in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0).
+
+## Enable client-side telemetry for web applications
+
+The preceding steps are enough to help you start collecting server-side telemetry. If your application has client-side components, follow the next steps to start collecting [usage telemetry](./usage-overview.md).
```

Comment:
"if your" - there is no need of this "if" here. The app is a well known app, and it has client-side.

Reply:
Fixed.